vm/devices/tpm/src/lib.rs

```diff
@@ -928,7 +937,8 @@ impl Tpm {
             if attestation_report_renew_elapsed > REPORT_TIMER_PERIOD
                 || self.attestation_report_renew_time.is_none()
             {
-                if let Err(e) = self.renew_attestation_report() {
+                // Renew tha attestation report as part of the request creation call.
```

Comment:
I'm not sure I fully understand the factoring, but having a comment that says we do a renewal as part of a call to a `create_` method, all the while there exists another method called `renew_` seems quite confusing.

something needs to be reworded / renamed / refactored here